src/main/scala/cromwell/util/EnhancedExponentialBackoff.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.util
+
+import com.google.api.client.util.ExponentialBackOff
+
+class EnhancedExponentialBackoff(enhancedBuilder: EnhancedExponentialBackoff.Builder) extends ExponentialBackOff(enhancedBuilder) {
+  private var firstCall: Boolean = true
```

Comment:
The Google one has a couple more features (including jiggling - which really is a one line method), but otherwise `retryBlock` is pretty much an `ExponentialBackoff`, with the addition that it actually executes something with exponential backoff, the Google just provides a "how much should I wait before trying again".